### PR TITLE
Improve description of external links in issue template dropdown

### PR DESCRIPTION
**Pull request by tobie**

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/config.yml [hunk 1]
```diff
@@ -2,9 +2,10 @@ blank_issues_enabled: false
 contact_links:
   - name: Report a Code of Conduct violation
     url: https://code-of-conduct.openjsf.org/
+    about: "Do you want to report a Code of Conduct violation?"
   - name: Legal Help
     url: https://github.com/openjs-foundation/cross-project-council#help-with-legal-questions-for-existing-projects-only
-    about: "Do you need help with legal matters?"
+    about: "Do your OpenJSF project need help with legal matters?"
```
[the remark this answers is not in the document]
Comment (tobie) [reply .github/ISSUE_TEMPLATE/config.yml]: Thanks. Had already fixed my bad grammar! 🙇

**Comment by tobie** [comment]
Magic incantation requiring fast track.

**Review by mhdawson — COMMENTED** [review]
LGTM

**Comment by tobie** [comment]
I have to look up the fast track process every. Single. Time.

**Pull request merged**